Source/WebCore/page/csp/ContentSecurityPolicy.cpp

```diff
@@ -888,7 +882,7 @@ void ContentSecurityPolicy::reportViolation(const String& effectiveViolatedDirec
 
     auto report = CSPViolationReportBody::createReportFormDataForViolation(info, usesReportTo, violatedDirectiveList.isReportOnly(), effectiveViolatedDirective, m_referrer, violatedDirectiveList.header(), blockedURI, httpStatusCode);
```

Comment:
This should probably be revised to use your 'populateReport' pattern, but obviously not needed for this change.